corehq/apps/app_manager/xpath_validator/js/xpathValidator.js

```diff
@@ -0,0 +1,11 @@
+global.SchemeNumber = require('js-xpath/lib/schemeNumber').SchemeNumber;
```

Comment:
I'm not sure how else I'd do this. I need the `SchemeNumber` object to be available to `'js-xpath/xpath'` (even though it doesn't explicitly `require` it). This line is basically making up for bad behavior on the part of the auto-generated xpath.js code.